webpack.config.js

```diff
@@ -0,0 +1,74 @@
+const path = require('path');
+const process = require('process');
+const webpack = require('webpack');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const { VueLoaderPlugin } = require('vue-loader');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const appConfig = {
+  entry: './src/index.ts',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist/public'),
+    publicPath: '/'
+  },
+  devtool: 'source-map',
+  devServer: {
+    host: process.env.TYPESCRIPT_DEMO_SPA_HOST,
+    port: process.env.TYPESCRIPT_DEMO_SPA_PORT,
+    watchContentBase: true,
+    historyApiFallback: true,
+    contentBase: [
+      path.resolve(__dirname, 'dist')
+    ],
+    publicPath: '/'
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js', '.vue', '.json'],
+    plugins: [new TsconfigPathsPlugin()]
+  },
+  module: {
+    rules: [
+      {
+        test: /\.vue$/,
+        loader: 'vue-loader'
+      },
+      {
+        test: /.tsx?$/,
+        loader: 'ts-loader',
+        options: {
+          appendTsSuffixTo: [/\.vue$/]
+        }
+      },
+      {
+        test: /\.css$/i,
+        use: [ MiniCssExtractPlugin.loader, 'css-loader' ]
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
```

Comment:
Do these cover all the image types that we use in the web? Probably not... but it's fine for now.

But what about font files?

Reply:
I just did some quick reading, it seems like local font files should work with `file-loader`, thought it would need some more config: 
https://chriscourses.com/blog/loading-fonts-webpack

Shall i add that in too?

Reply:
You did already.